website/static/js/pages/login-page.js

```diff
@@ -1,6 +1,25 @@
 /**
  * Login page
  */
-var LogInForm = require('js/signIn');
+var $ = require('jquery');
 
+var LogInForm = require('js/signIn');
 new LogInForm.SignIn('#logInForm');
+
+var SignUp = require('../signUp.js');
+new SignUp('#signUpScope', '/api/v1/register/', $('#campaign').val());
```

Comment:
As discussed via flowdock, we normally use `window.contextVars` to pass variables (like URLs) to JS files. 

(This comment accidentally got lost on an "outdated" diff but probably still applies)